tests/manager/core/test_opds_format_priority.py

```diff
@@ -0,0 +1,356 @@
+from __future__ import annotations
+
+import json
+from collections.abc import Callable
+from unittest.mock import MagicMock
+
+import pytest
+
+from palace.manager.core.opds_format_priority import (
+    FormatPriorities,
+    FormatPrioritiesSettings,
+)
+from palace.manager.sqlalchemy.model.licensing import (
+    DeliveryMechanism,
+    LicensePoolDeliveryMechanism,
+)
+
+
+class TestFormatPriorities:
```

Comment:
This is moved from `test_licensing.py`, since it didn't seem like it belonged there in the first place.

Its mostly copied, there are a couple minor changes to the test because `FormatPriorities` no longer filters out mechanisms.